eslint.config.mjs

```diff
@@ -0,0 +1,230 @@
+      '@typescript-eslint/explicit-member-accessibility': 'off',
+      '@typescript-eslint/explicit-module-boundary-types': 'off',
+      '@typescript-eslint/indent': 'off',
+      '@typescript-eslint/no-duplicate-enum-values': 'off',
+      '@typescript-eslint/no-empty-function': 'off',
+      '@typescript-eslint/no-empty-interface': 'off',
+      '@typescript-eslint/no-explicit-any': 'off',
+      '@typescript-eslint/no-inferrable-types': 'off',
+      '@typescript-eslint/no-misused-new': 'error',
+      '@typescript-eslint/no-namespace': 'error',
+
+      '@typescript-eslint/no-unused-vars': [
+        'error',
+        {
+          argsIgnorePattern: '^_'
+        }
+      ],
+
+      '@typescript-eslint/no-use-before-define': 'off',
+      '@typescript-eslint/no-var-requires': 'off',
+      '@typescript-eslint/prefer-for-of': 'error',
+      '@typescript-eslint/prefer-function-type': 'error',
+      '@typescript-eslint/prefer-namespace-keyword': 'error',
+      '@typescript-eslint/unified-signatures': 'error',
+      '@typescript-eslint/explicit-function-return-type': 'off',
+      'arrow-body-style': 'error',
+
+      camelcase: [
+        'error',
+        {
+          ignoreDestructuring: true
+        }
+      ],
+
+      'constructor-super': 'error',
+      curly: 'error',
+      'dot-notation': 'error',
+      eqeqeq: ['error', 'smart'],
+      'guard-for-in': 'error',
+      'max-classes-per-file': ['error', 1],
+      'max-len': 'off',
+      'no-nested-ternary': 'error',
+      'no-bitwise': 'error',
+      'no-caller': 'error',
+      'no-cond-assign': 'error',
+      'no-console': 'error',
+      'no-debugger': 'error',
+      'no-empty': 'error',
+      'no-eval': 'error',
+      'no-new-wrappers': 'error',
+      'no-prototype-builtins': 'off',
+      'no-shadow': 'off',
+      'no-throw-literal': 'error',
+      'no-trailing-spaces': 'off',
+      'no-undef-init': 'error',
+      'no-unsafe-finally': 'error',
+
+      'no-unused-expressions': [
+        'error',
+        {
+          allowTernary: true,
+          allowShortCircuit: true
+        }
+      ],
+
+      'no-unused-labels': 'error',
+      'no-var': 'error',
+      'object-shorthand': 'error',
+      'one-var': ['error', 'never'],
+      'patternfly-react/import-tokens-icons': 'error',
+      'patternfly-react/no-anonymous-functions': 'error',
+      'prefer-const': 'error',
+      'prettier/prettier': 'error',
+      radix: ['error', 'as-needed'],
+      'react/prop-types': 0,
+      'react/display-name': 0,
+      'react-hooks/exhaustive-deps': 'warn',
+
+      'react/no-unescaped-entities': [
+        'error',
+        {
+          forbid: ['>', '}']
+        }
+      ],
+
+      'spaced-comment': 'error',
+      'use-isnan': 'error',
+      'patternfly-react/no-layout-effect': 'error',
+      'valid-typeof': 'off'
+    }
+  },
+  {
+    files: ['**/examples/*', '**/demos/examples/**/*'],
+
+    rules: {
+      'patternfly-react/no-anonymous-functions': 'off'
```

Comment:
This should be the line that is disabling the displayName rule for examples. I'm unsure why it's still applying to just the chart examples.

My thinking is that maybe since the eslint.config-md.mjs is being run after the eslint.config.mjs, it re-applies the rule that was excluded?